### PR TITLE
Use Nashorn for CommonMark rendering

### PR DESCRIPTION
**Pull request by seanf**

### CODE REVIEW

File: zanata-war/src/main/java/org/zanata/util/CommonMarkRenderer.java [hunk 1]
```diff
@@ -21,20 +21,22 @@
 package org.zanata.util;
 
 import com.google.common.base.Throwables;
+import com.google.common.io.Resources;
+import jdk.nashorn.api.scripting.JSObject;
```
Comment (carlosmunoz) [inline zanata-war/src/main/java/org/zanata/util/CommonMarkRenderer.java]: @seanf Is this an OpenJDK-specific API? If not, is there a fallback?

Comment (seanf) [inline zanata-war/src/main/java/org/zanata/util/CommonMarkRenderer.java]: Nashorn is part of both Oracle JDK and OpenJDK 1.8.  It's unfortunate that we have to depend on a Nashorn interface here, but there seems to be no way around that if we want the best performance.

Comment (seanf) [inline zanata-war/src/main/java/org/zanata/util/CommonMarkRenderer.java]: Nashorn is also in Azul Zulu 8 and IBM's JDK 8.

**Comment by carlosmunoz** [comment]
:+1:

**Comment by seanf** [comment]
For testing, please note that we currently use server-side javascript:

* when rendering markdown on the home page
* when rendering markdown on a project About page

**Comment by djansen-redhat** [comment]
:white_check_mark: This looks fine to me

**Pull request merged**